### ISSUE TITLE
Accept transfer button missing text alternative

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6770]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the `Accept` button in the `Transfers Received` box does not have a text alternative per WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

Prereq: make sure that another user has transferred a work to you, but that you have not yet accepted the transfer request.

1. In Chrome, download the Site Improve browser extension
2. Go to https://dev.nurax.samvera.org/dashboard/transfers?locale=en and run the Site Improve browser extension

![text_alternative_button](https://github.com/samvera/hyrax/assets/32644254/d43d077a-4ffa-4838-9d9c-ebd70c6ee7ab)

**Comment by christhepianist** [comment]
Tested and confirmed that this is no longer an issue.
